player_test.py

```diff
@@ -146,16 +146,17 @@ def test_same_hand_gives_different_scores_for_different_anthill_configurations(s
     self.assertEqual(mario.score, expected_score_a)
     self.assertEqual(luigi.score, expected_score_b)
 
-class ChooseAntToMoveTest(unittest.TestCase):
+class MakeChoiceTest(unittest.TestCase):
   def test_returns_user_input_for_input_is_red(self):
   # test 37
-    mario = Player("placeholder name")
+    mario = Player("mario")
     allowed_choices = ['red', 'yellow', 'green', 'purple', 'brown']
+    prompt_text = "please enter your choice of ant"
     expected_result = "red"
     input_patcher = mock.patch('builtins.input', return_value = "red")
     InputMock = input_patcher.start()
```

Comment:
I think this should use snake case, i.e. `input_mock` because of this example I found:
```
>>> patcher = patch('__main__.thing', first='one', second='two')
>>> mock_thing = patcher.start()
>>> mock_thing.first
'one'
>>> mock_thing.second
'two'
```
https://docs.python.org/3/library/unittest.mock.html